### ISSUE TITLE
Make Git URL configurable

### ISSUE THREAD

**Issue by berendt** [SovereignCloudStack/k8s-cluster-api-provider#237]
```
terraform/files/bin/get_k8s_git.sh: git clone https://github.com/SovereignCloudStack/k8s-cluster-api-provider || exit 1
```

It should be possible to use another source here. Therefore, this should be configurable via a parameter.

**Comment by garloff** [comment]
So you want to be able to point this to a mirror or clone?
It's not hard to parametrize this, indeed.

**Comment by garloff** [comment]
Addressed by #266 and #268
